src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
```

Comment:
would be nice to add a `README.md` file with a simple example similar to the other components 🙂

Reply:
@Prokyonn I think this is not added yet, right?